modules/apis/pages/api-operations.adoc

```diff
@@ -1,19 +1,19 @@
 = API operations
 
 //TODO: Needs more work and clean up of all source blocks.
-This {product} workbook is a comprehensive guide that provides detailed examples and  practices for managing the {product} platform using the DevOps APIs.
+This {product} workbook is a comprehensive guide that provides detailed examples and practices for managing the {product} platform using the {devops-api}s.
```

Comment:
side question: do we translate our docs? At GitHub, we had to have a variable like `{devops-api-plural}` so that the translation service would not try to translate pluralized variables.

Reply:
We don't do translation (yet). I assume this would be something we would do if we were going to prepare the docs for translation.